BUILD.md

```diff
@@ -24,9 +24,8 @@ In order to build the application on Windows for the `Release` configuration, ru
 > git clone https://github.com/systelab/cpp-rest-api-core
 > md build && cd build
 > conan remote add systelab-public https://systelab.jfrog.io/artifactory/api/conan/cpp-conan-production-local
-> conan install .. -s build_type=Release -s compiler.toolset=v142 -s arch=x86_64
-> cmake .. -G "Visual Studio 16 2019" -A x64
-> "$VSINSTALLPATH/devenv.com" RESTAPICore.sln /build "Release" /PROJECT "RESTAPICore"
+> conan install . --profile=vs2022.conanprofile -s build_type=Release -s arch=x86_64 -if build/Release_x64
```

Comment:
For readability I suggest using `--install-folder` instead of `-if` and `--build-folder` instead of `-bf`

Reply:
sounds good, I'll modify it